lib/inv_cg_quda.cpp

```diff
@@ -133,7 +133,7 @@ namespace quda {
   }
 
   CGNR::CGNR(DiracMatrix &mat, DiracMatrix &matSloppy, SolverParam &param, TimeProfile &profile) :
-    CG(mdagm, mdagmSloppy, param, profile), mdagm(mat.Expose()), mdagmSloppy(mat.Expose()),
+    CG(mdagm, mdagmSloppy, param, profile), mdagm(mat.Expose()), mdagmSloppy(matSloppy.Expose()),
     bp(nullptr), init(false) {
```

Comment:
clang-format suggested the following change:


```
-  CGNR::CGNR(DiracMatrix &mat, DiracMatrix &matSloppy, SolverParam &param, TimeProfile &profile) :
-    CG(mdagm, mdagmSloppy, param, profile), mdagm(mat.Expose()), mdagmSloppy(matSloppy.Expose()),
-    bp(nullptr), init(false) {
+  CGNR::CGNR(DiracMatrix& mat, DiracMatrix& matSloppy, SolverParam& param, TimeProfile& profile)
+      : CG(mdagm, mdagmSloppy, param, profile)
+      , mdagm(mat.Expose())
+      , mdagmSloppy(matSloppy.Expose())
+      , bp(nullptr)
+      , init(false)
+  {
```

<!-- identifier: clang-format -->
<!-- File: lib/inv_cg_quda.cpp -->